build.gradle

```diff
@@ -22,6 +22,7 @@ dependencies {
     implementation 'org.projectlombok:lombok:1.18.16'
     implementation 'ch.qos.logback:logback-core:1.2.3'
     implementation 'org.slf4j:slf4j-api:1.7.30'
+    implementation 'com.google.guava:guava:30.0-jre'
```

Comment:
For some reason I get exception caused by lack of Immutable list method form guava. That is why I added it again.